app/helpers/fhir_helper.rb

```diff
@@ -146,7 +141,11 @@ def patient_as_fhir(patient)
         to_datetime_extension(patient.last_assessment_reminder_sent, 'last-assessment-reminder-sent'),
         to_bool_extension(patient.pause_notifications, 'paused-notifications'),
         to_string_extension(patient.status_as_string, 'status'),
-        to_bool_extension(patient.user_defined_symptom_onset, 'user-defined-symptom-onset')
+        to_bool_extension(patient.user_defined_symptom_onset, 'user-defined-symptom-onset'),
+        to_string_extension(patient.alternate_contact_type, 'alternate-contact-type'),
+        to_string_extension(patient.alternate_contact_name, 'alternate-contact-name'),
```

Comment:
Sorry sent last comment too soon. Since we are currently capturing the info in `Patient.telecom`, shouldn't it be moved to `Patient.contact`? That they would also eliminate the need for the `alternate-contact-type` and `alternate-contact-name` extensions.

This also raises the issue of how to handle primary contact if not Self. I think the most proper thing to do would be to use `telecom` only if primary contact type is Self and use `contact` otherwise. This would also eliminate the need for the `contact-type` and `contact-name` extensions.

Reply:
Oh yeah true. We should definitely be using `Patient.contact`. I think what you're saying with only using `telecom` when primary contact is `Self` makes sense. I will update it.

Reply:
Ok @tstrass, the latest 3 commits should update this to use `Patient.contact`. It ended up being more involved than I would've liked, so @timwongj and @sgober I'm going to have to re-request review.

Since I'm departing soon I wanted to get this up before I left, but it was a bit of a crunch to get this done, so please review carefully, since I'm not as confident as I'd like to be that I did not miss something.